tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "compilerOptions": {
+    "strict": true,
+    "noEmit": true
+  },
+  "include": ["test/**/*.ts"],
```

Comment:
This tells TypeScript to specifically try to compile the test files, and fails if the test files contain type errors